src/commands/gate/renderer.ts

```diff
@@ -82,13 +83,29 @@ export const renderGateEvaluationInput = (evaluateRequest: Payload): string => {
   return fullStr
 }
 
-export const renderGateEvaluationError = (error: any): string => {
-  let fullStr = chalk.red('ERROR: Could not evaluate the rules.')
-  if (error.response) {
-    fullStr += chalk.red(` Error Status: ${error.response.status}`)
+export const renderGateEvaluationError = (error: any, failIfUnavailable: boolean): string => {
+  let errorStr = 'ERROR: Could not evaluate the rules.'
+  if (!error.response) {
+    return chalk.red(`${errorStr}\n`)
   }
 
-  fullStr += '\n'
+  errorStr += ` Status code: ${error.response.status}.\n`
+  if (isBadRequestError(error)) {
```

Comment:
is this condition only valid for HTTP 400 errors? No other 4xx?

Reply:
Correct, we return 400 in case of validation failures and only in those cases we should convey the error message to the user